### PR TITLE
CP-41118:Replace Citrix Tradmark

### PR DESCRIPTION
**Pull request by MinghuiHu**

### CODE REVIEW

**Comment by stormi** [comment]
Ideally, this PR should be rebase on latest master and contain only one commit. Note: this can be done without closing it and creating a new one. All you need it managing the rebase and then doing a force push of your branch.

If the PR was merged the way it would create a rather confusing commit history.

**Comment by xihuan-citrix** [comment]
> Ideally, this PR should be rebased on latest master and contain only one commit. Note: this can be done without closing it and creating a new one. All you need it managing the rebase and then doing a force push of your branch.
> 
> If the PR was merged the way it is, this would create a rather confusing commit history.

Hi @MinghuiHu, could you help to merge all the commits?

File: mk/xe-linux-distribution [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 
-# Copyright (c) 2015, Citrix Systems
+# Copyright (c) 2015, Cloud Software Group Holdings
```
[the remark this answers is not in the document]
Comment (xihuan-citrix) [reply mk/xe-linux-distribution:3]: We need to modify this year as well.

[the remark this answers is not in the document]
Comment (MinghuiHu) [reply mk/xe-linux-distribution]: I have updated the year to 2022

[the remark this answers is not in the document]
Comment (MinghuiHu) [reply mk/xe-linux-distribution]: > We need to modify this year as well.

I have updated the year to 2022

File: mk/debian/control [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Source: xe-guest-utilities
 Section: main/admin
 Priority: optional
-Maintainer: Citrix Systems, Inc. <www.citrix.com>
+Maintainer: Cloud Software Group Holdings, Inc. <www.cloud.com>
```
[the remark this answers is not in the document]
Comment (MinghuiHu) [reply mk/debian/control]: > `Cloud Software Group` is enough, remove `<www.citrix.com>`

both fixed

**Review by xihuan-citrix — APPROVED** [review]

**Review by liulinC — APPROVED** [review]

**Pull request merged**